package.json

```
@@ -23,7 +23,8 @@
     "lint": "npm run lint:polymer && npm run lint:eslint && npm run lint:prettier",
     "win-lint": "npm run win-lint:polymer && npm run lint:eslint && npm run lint:prettier",
     "format": "npm run format:polymer && npm run format:prettier && npm run format:eslint",
-    "win-format": "npm run win-format:polymer && npm run format:prettier && npm run format:eslint "
+    "win-format": "npm run win-format:polymer && npm run format:prettier && npm run format:eslint ",
+    "storybook": "cd storybook && npm start"
```

I guess this will throw an error if docs were never run before, since there won't be an `analysis.json`:
```suggestion
    "storybook": "lerna run analysis --parallel && cd storybook && npm start"
```

---

I had it at some point but the analysis takes quite a while and I felt like it would be very cumbersome to keep executing it. I'll have another go at it though 👍